### DISCUSSION TITLE
HasIndex这个FluentApi操作被删掉了？

### DISCUSSION THREAD

**Discussion by coolqingcheng** [dotnetcore/FreeSql#1408] [General]
目前最新版只找到Index方法。但是Index不支持(a=>new {field}这种操作。
写字符串的话，容易搞错，

希望恢复        eb.HasIndex(a => a.Title).IsUnique().HasName("idx_tb_song1111");

**Comment by 2881099** [comment]
EF 风格的 API 都封装在 FreeSql.DbContext.dll 里面
